play.py

```diff
@@ -0,0 +1,53 @@
+import argparse
+import json
+
+from api import Api
+from api_wrapper import api_wrapper_test
+import prerechnung
+import PySimpleGUI as sg
+
+
+def arg_parser():
```

Comment:
we can add `play.py` for illustrative and experimental purposes. But then the code duplication with `erpnext.py` should be eliminated. So please move this code e.g. to two functions `arg_parser` and `init` in a new `args.py` and import the functions in both `play.py` and `erpnext.py`. Note that then the longer function `arg_parser` from `erpnext.py` should be used, augmented with the new `--google-josn` option.